README.md

```diff
@@ -70,6 +70,7 @@ Creates a new bossbat instance. All arguments are optional.
 - `options.connection`: Used to configure the connection to your redis. This accepts the same arguments that [`ioredis`](https://github.com/luin/ioredis/blob/master/API.md#new_Redis_new) does.
 - `options.prefix`: A string that all redis keys will be prefixed with. Defaults to `bossbat`.
 - `options.ttl`: The number of milliseconds before a job times out. Setting it will change the maximum duration that jobs can hold a lock. By default, job locks will timeout if a job does not complete in 2000ms.
+- `options.tz`: An optional timezone used with `jobDefinition.cron` expressions.
```

Comment:
The use case for this is that I want the cron expressions to be in the same timezone with my development team. Thus we don't have to worry in which location/timezone the software is deployed at.

In my opinion it would be confusing if different expressions could have different timezones. In this way setting it once in the Bossbat options is more sensible.